lib/src/widgets/chat_groupedlist_widget.dart

```diff
@@ -176,65 +158,71 @@ class _ChatGroupedListWidgetState extends State<ChatGroupedListWidget>
+        physics: showPopUp ? const NeverScrollableScrollPhysics() : null,
+        controller: widget.scrollController,
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            GestureDetector(
+              onHorizontalDragUpdate: (details) => isEnableSwipeToSeeTime
+                  ? showPopUp
+                      ? null
+                      : _onHorizontalDrag(details)
+                  : null,
+              onHorizontalDragEnd: (details) => isEnableSwipeToSeeTime
+                  ? showPopUp
+                      ? null
+                      : _animationController?.reverse()
+                  : null,
+              onTap: widget.onChatListTap,
+              child: _animationController != null
+                  ? AnimatedBuilder(
+                      animation: _animationController!,
+                      builder: (context, child) {
+                        return _chatStreamBuilder;
+                      },
+                    )
+                  : _chatStreamBuilder,
             ),
-          if (chatController != null)
-            Flexible(
-              child: Align(
-                alignment: suggestionsListConfig.axisAlignment.alignment,
-                child: ValueListenableBuilder(
-                  valueListenable: chatController!.newSuggestions,
-                  builder: (context, value, child) {
-                    return SuggestionList(
-                      suggestions: value,
-                    );
-                  },
+            if (chatController != null)
+              ValueListenableBuilder(
+                valueListenable: chatController!.typingIndicatorNotifier,
+                builder: (context, value, child) => TypingIndicator(
+                  typeIndicatorConfig: widget.typeIndicatorConfig,
+                  chatBubbleConfig: chatBubbleConfig?.inComingChatBubbleConfig,
+                  showIndicator: value,
+                ),
+              ),
+            if (chatController != null)
+              Flexible(
+                child: Align(
+                  alignment: suggestionsListConfig.axisAlignment.alignment,
+                  child: ValueListenableBuilder(
+                    valueListenable: chatController!.newSuggestions,
+                    builder: (context, value, child) {
+                      return SuggestionList(
+                        suggestions: value,
+                      );
+                    },
+                  ),
                 ),
               ),
-            ),
 
-          // Adds bottom space to the message list, ensuring it is displayed
-          // above the message text field.
-          SizedBox(
-            height: chatTextFieldHeight,
-          ),
-        ],
+            // Adds bottom space to the message list, ensuring it is displayed
+            // above the message text field.
+            SizedBox(
+              height: MediaQuery.of(context).size.height,
```

Comment:
This should be from the parameter as to ensure consistency between the value used in chat_view.dart and here.